src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -18,18 +18,16 @@ object JSONUtil {
 
   private def toJSONFromNode(node: Node): String = {
     val sb = new StringBuilder
-
+    val escapedsb = new StringBuilder
+    val returnsb = new StringBuilder
     sb.append('{')
     sb.append(List("\".tag\":\"",node.label,"\",").mkString)
     sb.append("\".child\":")
     sb.append("[")
-    
     sb.append(serializing(node))
-
     sb.append("]")
     sb.append("}")
-
-    pretty(render(parse(sb.toString)))
+    pretty(render(parse(sb.toString.replace("\\","\\\\"))))
```

Comment:
Could you add a comment above this line why this replace method is necessary, possibly by pointing to the URL of the issue?